index.html

```diff
@@ -1,227 +1,249 @@
-              <ul class="list-stack">
-                <li>html</li>
-                <li>css</li>
-                <li>javascript</li>
-              </ul>
-              <button type="button" class="btn-primary">See Project</button>
-            </div>
-          </div>
-
-
-          <div class="flex-container">
-            
-            <div class="flex-items">
-              <h2>Uber Navigation</h2>
-              <span class="subtitle">Uber </span>  
-              <span class="mid-list"> 
-                <span class="dot"></span>Lead Developer  
-                <span class="dot"></span>2018
-              </span>
-
-              <p class="description">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
-              <ul class="list-stack">
-                <li>html</li>
-                <li>css</li>
-                <li>javascript</li>
-              </ul>
-              <button type="button">See Project</button>
-            </div>
-            <div class="flex-items"><img src="images/Snapshoot_PortfolioD.svg" alt="Portfolio" /></div>
-          </div>
-      </section>
-  
-    <!--This section is the ENDS WORKS AREA-->
-
-
-    <!-- This is the section of ABOUT ME-->
-
-      <section>
-        <div class="main-container">
-          <div class="main-about-container">
-            <div class="flex-item item1"> 
-              <h2>About Myself</h2>
-              <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-              <h3>LET’S CONNECT</h3>
-              <ul class="smedia">
-                <li><a href="#"><img src="images/twitter.svg" alt="twitter" /></a></li>
-                <li><a href="#"><img src="images/Vector.png" alt="linkedin" /></a></li>
-                <li><a href="#"><img src="images/medium.svg" alt="medium" /></a></li>
-                <li><a href="#"><img src="images/github.svg" alt="github" /></a></li>
-                <li><a href="#"><img src="images/angellist.svg" alt="angellist" /></a></li>
-              </ul>
-              <button class="btn" type="button">Get My Resume</button>
-            </div>
-            <div class="flex-item item2">
-              <ul class="tabs">
-                <li>Languages <i class="pull-right fa fa-angle-down"></i>
-                  <div class="inner-container">
-                      <!-- <img src="images/Frame_9.svg" alt="javascript" />
+        <div class="flex-item item2">
+          <ul class="tabs">
+            <li>Languages <i class="pull-right fa fa-angle-down"></i>
+              <div class="inner-container">
+                <!-- <img src="images/Frame_9.svg" alt="javascript" />
```

Comment:
- [ ] Kindly when you changed your code I think you made a mistake with your `about` section because there's a big horizontal scrolling on your page kindly fix it so I can reach your menu.

![image](https://user-images.githubusercontent.com/35535130/167514922-9a0f17f6-3458-4a74-bfbd-cfdf8880608a.png)